drivers/bridge/bridge.go

```diff
@@ -147,6 +147,27 @@ func (c *networkConfiguration) Validate() error {
 	return nil
 }
 
+func (c *networkConfiguration) copyDefaultConfigs(option options.Generic, config *networkConfiguration) {
```

Comment:
Right. Then we need to do it after inside the parsing function where we already accessed the genricData options. In the same function before calling Validate().